services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeController.kt

```diff
@@ -142,6 +251,18 @@ class HankeController(
 
     @DeleteMapping("/{hankeTunnus}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
+    @Operation(summary = "Delete hanke", description = "Delete an existing hanke.")
```

Comment:
This should mention that deleting is not possible, if there are any active applications for the hanke.

Reply:
Description updated.